app/models/keyword_upload_file.rb

```diff
@@ -0,0 +1,37 @@
+class KeywordUploadFile
+  include ActiveModel::Model
+
+  attr_accessor :file
+
+  validates :file, presence: true
+  validate :keywords_number_in_file
+
+  def initialize(file: nil)
+    @file = file
+  end
+
+  def encoded_file
+    @encoded_file ||= file.read.force_encoding('UTF-8')
+  rescue StandardError
+    errors.add(:file, 'File cannot be encoded to UTF-8')
+  end
+
+  def words
+    @words ||= encoded_file.split(',').map(&:strip)
+  rescue StandardError
+    errors.add(:file, 'File cannot be split to words')
+  end
+
+  private
+
+  def keywords_number_in_file
+    maximum_keywords_per_upload = ENV.fetch('MAXIMUM_KEYWORDS_PER_UPLOAD', 100).to_i
+    minimum_keywords_per_upload = ENV.fetch('MINIMUM_KEYWORDS_PER_UPLOAD', 1).to_i
```

Comment:
The check on the minimum count might not require an env variable. The file should just not be empty 💡